src/plugins/intel_gpu/src/kernel_selector/core/actual_kernels/roi_align/roi_align_kernel_ref.cpp

```diff
@@ -76,15 +76,32 @@ bool ROIAlignKernelRef::Validate(const Params& p, const optional_params& o) cons
     return true;
 }
 
-JitConstants ROIAlignKernelRef::GetJitConstants(const roi_align_params &params) const {
+JitConstants ROIAlignKernelRef::GetJitConstants(const roi_align_params& params) const {
     JitConstants jit = MakeBaseParamsJitConstants(params);
+
     jit.AddConstant(MakeJitConstant("SPATIAL_SCALE", params.spatial_scale));
     jit.AddConstant(MakeJitConstant("SAMPLING_RATIO", params.sampling_ratio));
-    if (params.mode == PoolType::MAX)
+
+    if (params.pooling_mode == PoolType::MAX)
         jit.AddConstant(MakeJitConstant("MAX_POOL", true));
-    else if (params.mode == PoolType::AVG)
+    else if (params.pooling_mode == PoolType::AVG)
         jit.AddConstant(MakeJitConstant("AVG_POOL", true));
+
+    if (params.aligned_mode == roi_aligned_mode::asymmetric) {
+        jit.AddConstant(MakeJitConstant("OFFSET_SRC", 0));
```

Comment:
shouldn't be here 0f? to potentially avoid unneeded casting in kernel

Reply:
Done